src/facade/reply_builder.cc

```diff
@@ -347,71 +346,51 @@ void RedisReplyBuilder::SendEmptyArray() {
   StartArray(0);
 }
 
-void RedisReplyBuilder::SendStringArr(absl::Span<const std::string_view> arr) {
-  if (arr.empty()) {
+void RedisReplyBuilder::SendStringArr(absl::Span<const std::string_view> arr, CollectionType type) {
+  if (type == ARRAY && arr.empty()) {
     SendRaw("*0\r\n");
     return;
   }
 
-  SendStringCollection(arr.data(), arr.size(), CollectionType::ARRAY);
+  SendStringCollection(arr, type);
 }
```

Comment:
Is there a reason we have this "*0\r\n" shortcut for the string-view version, but not for the string version? Seems like not